### ISSUE TITLE
Force restart NVDA after updating

### ISSUE THREAD

**Issue by haruncetinkaya306** [nvaccess/nvda#16229]
### Is your feature request related to an issue? Please define.
As it is known, various error reports are reported by many users after updating.
### Describe your desired solution
After updating NVDA, an option to restart the computer should be added as a feature. This option should only be in beta/rc channel and stable channel.
### Describe the alternatives you considered
None.
### Additional context

**Comment by Brian1Gaff** [comment]
Would this be wise, as it may be some documents and processes might still be running at that time. Normally the funny effects are behaviour changing but not disabling. I'd prefer, restart notw yess no. Brian

-- 
***@***.*** 
Sent via blueyonder.(Virgin media) 
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
  ----- Original Message ----- 
  From: haruncetinkaya306 
  To: nvaccess/nvda 
  Cc: Subscribed 
  Sent: Monday, February 26, 2024 6:58 PM
  Subject: [nvaccess/nvda] Force restart NVDA after updating (Issue #16229)



  Is your feature request related to an issue? Please define.
  As it is known, various error reports are reported by many users after updating.

  Describe your desired solution
  After updating NVDA, an option to restart the computer should be added as a feature. This option should only be in beta/rc channel and stable channel.

  Describe the alternatives you considered
  None.

  Additional context

**Comment by XLTechie** [comment]
Forcing is never going to happen. Offering the option to restart now, to help 
encourage it though, that might happen.

I'm actively working on this area, I might include a "Restart now" button in an 
upcoming pull request.
